documents/policies/external/Competition Requirements.md

```diff
@@ -97,20 +98,38 @@ The WCA Delegate must submit the following information when requesting approval
    2. If the competitor limit for a competition or an event is reached before the competition, the organization team must not approve any additional registrations for the competition or the event.
-   3. The competitor limit is per competition. Per day competitor limits are not allowed.
+   3. The competitor limit is per competition or per event. Per day competitor limits are not allowed.
+      1. Exception: for 3x3x3 Fewest Moves simultaneous competitions, the competitor limit is per location.
    4. The WCA Competition Announcement Team will review the competitor limit and/or qualification results or rankings to ensure that they are utilized properly.
 6. Registration requirements
    1. The registration requirements must be clear and precise.
       1. The registration requirements must be available in English.
       2. In addition to the English version, the registration requirements can be available in any other language and should be available in the local language.
+      3. If the registration requirements are not possible to be fulfilled by an individual due to reasons outside of their control, they are eligible for completing their registration following a suitable alternative approach, at the discretion of the Delegate. In case of such an inquiry, organizers must involve the Delegate, who must then carefully assess the individual's situation. If deemed appropriate, the Delegate and organizers must make reasonable efforts to provide an alternative approach for the individual to fulfilling the requirements.
+         1. Possible reasons include, but are not limited to:
+            1. Having an unsupported credit card for paying the registration fee.
+         2. Exceptions must not be based on:
+            1. Missing the registration deadline.
+            2. Not having enough money to pay the registration fee.
    2. The registration period(s) must be a clearly defined time range with a start and end time and date.
       1. Each registration period must be announced publicly on the WCA website at least 48 hours before it starts.
          1. The competition's WCA Delegate(s) and/or organizer(s) may register for the competition prior to the first registration period, however their registration(s) must not be approved until the first registration period has started.
       2. Different registration periods may have different registration requirements (e.g. restricting which events a competitor can register for).
-   3. Different locations for multi-location competitions may have different registration requirements (e.g. competitor limit).
+   3. The deadline for updating registered events for competitors with completed registrations must be outlined.
+      1. This deadline will either be at the same time as registration closes or after registration has closed.
+         1. The deadline may be set to be just before an event is scheduled to take place. 
```

Comment:
I would rather propose to add this to the top-level paragraph instead as default if no other point in time specified.

Reply:
Personally, I see it as being the default that people cannot change their registration after registration closes due to the implications it may have.  That is why I added it as a clarifying point.

Reply:
Ok, let's see how this works out in practice.